openresty-patches/patches/1.15.8.3/LuaJIT-2.1-20190507_03-dyn-lightuserdata-map.patch

```diff
@@ -113,26 +113,26 @@ index a0fba2a..1ab3097 100644
 +
 +static char checked_layout = 0;
 +
-+#define LJ_LUV_SECT_MASK (7UL << LJ_LUV_SECT_SHIFT)
-+uint8_t luv_sect_map[8] = {
++#define LJ_LUV_NUM_SECTS (1 << LJ_LUV_SECT_BITS)
```

Comment:
Move this to `lj_arch.h` so it can be used in other places.

Reply:
done